experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -600,6 +600,65 @@ export interface Sequence<in out TTypes extends AllowedTypes> extends TreeField
 	 */
 	removeRange(start?: number, end?: number): void;
 
+	/**
+	 * Moves the specified item to the start of the sequence.
+	 * @param sourceIndex - The index of the item to move.
+	 * @throws Throws if `sourceIndex` is invalid.
+	 * @remarks
+	 * All indices are relative to the sequence excluding the nodes being moved.
```

Comment:
Does this comment really make sense here?

Reply:
Fixed.